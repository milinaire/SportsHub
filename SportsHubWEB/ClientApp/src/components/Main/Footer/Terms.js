import React, { Component, Fragment } from "react";
import "./Footer.css";
import {BaseLayout} from "../Layout/BaseLayout";


export class Terms extends Component {
    state={
        Headline:"Terms",
        Text:"Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, "
    }
  render() {
    return (
      <Fragment>
        <BaseLayout>
          <div>
            <h1>{this.state.Headline}</h1>
            <p>{this.state.Text}</p>
          </div>
        </BaseLayout>
      </Fragment>
    );
  }
}
