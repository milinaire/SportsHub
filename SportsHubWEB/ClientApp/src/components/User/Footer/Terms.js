import React, {Component, Fragment} from "react";
import "./Footer.css";


export class Terms extends Component {
  state = {
    Headline: "Terms",
    Text: "Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, "
  }

  render() {
    return (
      <Fragment>
        <div className="main-part-footer">
          <h1>{this.state.Headline}</h1>
          <p>{this.state.Text}</p>
        </div>
      </Fragment>
    );
  }
}
