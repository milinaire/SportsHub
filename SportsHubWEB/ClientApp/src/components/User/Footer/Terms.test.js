
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import {Terms} from "./Terms";


describe('Main\Footer\Terms test', () => {
  it('Main\Footer\Terms should match snapshot', () => {
    const component = renderer.create(<Terms/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
