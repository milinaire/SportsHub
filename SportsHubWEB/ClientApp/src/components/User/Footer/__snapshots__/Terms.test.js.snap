// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooterTerms test MainFooterTerms should match snapshot 1`] = `
<div
  className="main-part-footer"
>
  <h1>
    Terms
  </h1>
  <p>
    Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, Many text from db, 
  </p>
</div>
`;
